packages/server/src/app/challenge/routers/challenge.router.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import { jwtAuthHeaderValidator } from '../../auth/index.js';
+import { userChallengeController } from '../controllers/challenge.controller.js';
+import { challengeSuggestionRouter } from './challenge_suggestion.router.js';
+
+export const challengeRouter = express.Router();
+
+challengeRouter.get('/', jwtAuthHeaderValidator, async (request, response) => {
```

Comment:
I may be misremembering, but I thought we had spoken about that GET `/api/challenge/` would return all challenges (more an API end point for managing the challenges), while GET `/api/user/:userId/challenge` would return all challenges from 1 particular user?